# std_msgs/float64 percep_rmse
bool ee_visible
bool probe_visible
bool pelvis_visible
bool robot_connected
